Service/AddOn/ReadmeBuilder.php

```diff
@@ -393,29 +434,26 @@ protected function generateHtml() : string
          * @param string $tableTitle
          * @param ArrayCollection|array|Entity[] $entities
          * @param array $headerMap
+         * @param bool $groupedEntities
          */
-        $generateTableFromEntity = function (string $tableTitle, $entities, array $headerMap) use(&$readme) : void
+        $generateTableFromEntity = function (string $tableTitle, $entities, array $headerMap, $groupedEntities = false) use(&$readme) : void
```

Comment:
Please type-hint `$entities` as ArrayCollection.

Reply:
I've type hinted `$groupedEntities`, but `$data['cli_commands']` is not an `ArrayCollection`, and type hinting `$entities` here will thus not work.